sdk/core/azure-core-tracing-opentelemetry/pom.xml

```diff
@@ -80,6 +80,50 @@
       <version>3.3.3</version> <!-- {x-version-update;org.mockito:mockito-core;external_dependency} -->
       <scope>test</scope>
     </dependency>
+
+    <!-- sample file dependencies -->
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-data-appconfiguration</artifactId>
+      <version>1.1.9</version> <!-- {x-version-update;com.azure:azure-data-appconfiguration;dependency} -->
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-sdk</artifactId>
+      <version>0.17.1</version> <!-- {x-version-update;io.opentelemetry:opentelemetry-sdk;external_dependency} -->
+      <scope>test</scope>
+    </dependency>
```

Comment:
This can be removed as OTel-sdk is already included as compile scope dependency above.

Reply:
yes, right. I think its added previously as a test dep.